src/VerticalDivider/VerticalDivider.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import cx from "classnames"
+import t from 'prop-types'
+import Box from "../Box/Box"
+import "./verticaldivider.css"
+import * as types from '../types'
+const VerticalDivider = ({ tone = "subtle", className, size, ...props }) => {
+  const classes = cx(className, `u-vertical-divider--${tone}`, {[`u-vertical-divider--height-${size}`]: size})
+  return <Box aria-hidden height={size} minHeight={size ? undefined : 'smallish'}className={classes} {...props} />
```

Comment:
check formatting here `minHeight={size ? undefined : 'smallish'}className={classes}`

Reply:
thanks, i also removed the above line that adds the u-vertical-divider class that isn't necessary since that's handled on the Box component